datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/KafkaBasedConnectorConfig.java

```diff
@@ -197,4 +204,8 @@ public long getProcessingDelayLogThresholdMillis() {
   public boolean getEnablePartitionAssignment() {
     return _enablePartitionAssignment;
   }
+
+  public String getCallbackStatusStrategy() {
```

Comment:
rename the method and the variable name to better describe the functionality. i.e. `getCallbackStatusStrategyFactory`